packages/app/src/components/SearchBar/index.js

```diff
@@ -66,21 +67,53 @@ const SearchBar = () => {
 
   return (
     <>
-      <form onSubmit={handleSubmit}>
-        <fieldset>
-          <input
-            type="text"
-            name="search"
-            placeholder="  Search"
-            data-id="search-input"
-            onChange={handleInputChange}
-            aria-label="search-input"
-          />
-          <div data-id="error-message">
-            {showErrorMessage ? <>Application Not Found</> : null}
-          </div>
-        </fieldset>
-      </form>
+      <section className="searchBarWrapper">
+        <div className="row">
+          <form
+            className="column column-40 column-offset-25"
+            onSubmit={handleSubmit}
+          >
+            <fieldset>
+              <div className="searchBarInput">
+                <input
+                  type="text"
+                  name="search"
+                  placeholder="  Search"
```

Comment:
What's the reason for the leading whitespace here?

Reply:
just following the example sheet